rllib/offline/estimators/fqe_torch_model.py

```diff
@@ -138,8 +144,10 @@ def train_q(self, batch: SampleBatch) -> TensorType:
                 state_keys = ["state_in_{}".format(i) for i in range(num_state_inputs)]
 
                 # Compute action_probs for next_obs as in FQE
-                all_actions = torch.zeros([minibatch.count, self.policy.action_space.n])
-                all_actions[:] = torch.arange(self.policy.action_space.n)
+                all_actions = torch.zeros(
+                    [minibatch.count, self.policy.action_space.n], dtype=int
+                )
+                all_actions[:] = torch.arange(self.policy.action_space.n, dtype=int)
                 next_action_prob = self.policy.compute_log_likelihoods(
```

Comment:
call this `next_action_logp` so that taking `exp` on it makes sense.

Reply:
I also highly recommend to use inline comments to describe the math operation that you do at every step. This will help you remember what your intensions were at each step.